generators/app/tasks/installDependencies.js

```diff
@@ -1,27 +1,19 @@
 const latestSemver = require('latest-semver');
 const semverRegex = require('semver-regex');
 
+const OPTIONAL_DEPENDENCIES = require('../constants').OPTIONAL_DEPENDENCIES;
+
 const runCommand = require('./runCommand');
 
 const DEPENDENCIES = [
-  'reselect',
-  'apisauce',
   'history',
   'i18next',
   'lodash',
   'postcss',
-  'prop-types',
   'react',
   'react-dom',
-  'react-redux',
   'react-router',
-  'react-router-dom',
-  'react-router-redux',
-  'redux',
-  'redux-beacon',
-  'redux-form',
-  'redux-thunk',
-  'seamless-immutable'
+  'react-router-dom'
```

Comment:
I'd add husky if as a dependency

Reply:
husky is already added as a non-optional devDependency